css/main.css

```diff
@@ -305,6 +305,68 @@ footer {
   transform: scale(var(--s));
 }
 
+.loaders {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 7rem;
+  min-height: 26rem;
+}
+
+.skeleton-loader {
+  position: absolute;
+  margin-top: 20px;
```

Comment:
There is mix of px and rem , could we align with one ?
It is recommended to go with rem :)
Have we checked the responsiveness with mobile?

Reply:
Okay I'll make the changes.
And yes, it is responsive, see this: 

https://user-images.githubusercontent.com/61474917/124388690-93d75280-dd01-11eb-98c4-340ba3d97664.mp4